guidelines/enhancement_template.md

```diff
@@ -461,6 +461,19 @@ Describe how to
   - Namespaces deletion will not delete all objects in etcd, leading to zombie
     objects when another namespace with the same name is created.
 
+#### Operating at scale
```

Comment:
This is nested under the "Operational Aspects of API Extensions" section. Is that intentional? I could also see this being at the same level, for example.